### PR TITLE
UI/LOOP-952/new-sensor-workflow

### PR DESCRIPTION
**Pull request by nhamming**
Includes common elements from the Dexcom new sensor workflow.

### CODE REVIEW

File: LoopKitUI/Views/GuideButtonStyle.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
+//
+//  GuideButtonStyle.swift
+//  LoopKitUI
+//
+//  Created by Pete Schwamb on 2020-03-04.
```
Comment (rickpasetto) [inline LoopKitUI/Views/GuideButtonStyle.swift:5]: "Created by Pete Schwamb"?   Not a problem, just wondering if you really meant to put "Nathaniel Hamming"?  Or did Pete make this?

**Review by rickpasetto — APPROVED** [review]
LGTM!

**Review by mpangburn — APPROVED** [review]
LGTM!

File: LoopKitUI/Views/GuideButtonStyle.swift (hunk 1 above)
Comment (nhamming) [reply LoopKitUI/Views/GuideButtonStyle.swift:5]: I stole it from Pete.

**Pull request merged**